src/angularComponent/ngcModifySuggestionsTemplateFactory.js

```diff
@@ -32,24 +32,35 @@ export default function ngcModifySuggestionsTemplateFactory($document, $template
       const itemChildrenEl = doc.createElement('div');
       itemChildrenEl.setAttribute('ng-repeat', 'suggestion in suggestion.children');
       itemChildrenEl.setAttribute('ng-include', `'${templateCacheName}'`);
-      if (itemEl.hasAttributes()) {
-        for (let i = 0; i < itemEl.attributes.length; i++) {
-          const attr = itemEl.attributes[i];
-          itemChildrenEl.setAttribute(attr.name, attr.value);
-        }
-      }
+
       itemEl.parentNode.appendChild(itemChildrenEl);
 
+      const itemHeader = element.querySelector(
+        'ngc-omnibox-suggestion-header, [ngc-omnibox-suggestion-header]'
+      );
+      if (itemHeader) {
+        itemHeader.setAttribute('suggestion', 'suggestion');
+        itemHeader.setAttribute('ng-attr-aria-selected', '{{suggestionItem.isHighlighted()}}');
+        itemHeader.setAttribute('ng-attr-aria-readonly',
+            '{{!suggestionItem.isSelectable() || undefined}}');
```

Comment:
Instead of checking for falsiness, we should do `suggestionItem === false` to enforce a default. In order for something to actually not be selectable, the app-maker has to explicitly use `false`.